operator/templates/stateful-set.yaml

```diff
@@ -585,11 +590,27 @@ spec:
         - name: bootstrap
           image: {{ $.Params.NODE_DOCKER_IMAGE }}
           imagePullPolicy: {{ $.Params.NODE_DOCKER_IMAGE_PULL_POLICY }}
+          securityContext:
+            capabilities:
+              add:
+                # TODO(mpereira): add an operator parameter for "memory lock" so
+                # that this capability and a respective Docker image with
+                # `setcap`ed binaries are conditionally used.
+                - IPC_LOCK
```

Comment:
@ANeumann82 why we need the `IPC_LOCK` in bootstrap?

Reply:
https://github.com/mesosphere/kudo-cassandra-operator/blob/3884e69e01c19ed80e01b0c69ee76572fb245e44/images/Dockerfile#L23

Because of this, basically. We give the java binary the capabilities, so when we try to run java (i.e. nodetool), and the container doesn't have the caps, it fails.